Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx

```diff
@@ -35,8 +35,8 @@ export default class AccessContainer extends React.Component<IProps, IState> {
   public async componentDidMount() {
       // are we getting the person's access or the team's?
       const accessFetchUrl = this.props.person
-          ? `/access/listassigned?personId=${this.props.person.id}&teamId=${this.props.person.teamId}`
-      : `/access/list?teamId=${this.context.team.id}`;
+          ? `/api/${this.context.team.name}/access/listassigned?personId=${this.props.person.id}&teamId=${this.props.person.teamId}`
+      : `/api/${this.context.team.name}/access/list?teamId=${this.context.team.id}`;
```

Comment:
Is this redundant? I think it might be better as `/api/${this.context.team.name}/access/list? and update the action to look for x.team.Name = Team instead of x.Team.Id. Same with listassigned passing both team.name and teamId.

Reply:
true